victorLi/quiz/q1.js

```diff
@@ -0,0 +1,35 @@
+//Given an array of positive numbers and a positive number ‘k,’ find the maximum sum of any contiguous subarray of size ‘k’.
+
+
+//Answer 3,40,70
+let x = [10,50,20,30,40,70,10]
+let y = [100,50,20,70,30,90,10]
+let z = [20,30,10,50,80,20,30,90]
+
+const maxSum = (array, k ) =>{
+ 
+  let begin = 0
+  let end = k-1
+  let maxSum = -1
+  let result = []
+ 
+  while( end !== array.length )
+  {
+    let tempSum = array[begin] + array[begin+1] + array[end]
+    if( tempSum > maxSum)
+    {
+      maxSum = tempSum
+      result = [ array[begin] , array[begin+1] , array[end]]
+    }
+     
+    end++
+    begin++
+  }
+  return result
```

Comment:
This function should return the maximum sum, not the subarray that gives you the maximum sum.